core/src/integration-test/java/com/scalar/db/storage/StorageColumnValueIntegrationTestBase.java

```diff
@@ -453,13 +453,13 @@ public void put_WithNullValues_ShouldPutCorrectly() throws ExecutionException {
 
     Put put =
         new Put(new Key(partitionKeyValue))
-            .withNullValue(COL_NAME1)
-            .withNullValue(COL_NAME2)
-            .withNullValue(COL_NAME3)
-            .withNullValue(COL_NAME4)
-            .withNullValue(COL_NAME5)
-            .withNullValue(COL_NAME6)
-            .withNullValue(COL_NAME7)
+            .withBooleanValue(COL_NAME1, null)
+            .withIntValue(COL_NAME2, null)
+            .withBigIntValue(COL_NAME3, null)
+            .withFloatValue(COL_NAME4, null)
+            .withDoubleValue(COL_NAME5, null)
+            .withTextValue(COL_NAME6, null)
+            .withBlobValue(COL_NAME7, (ByteBuffer) null)
```

Comment:
I changed how to specify NULL values in this PR. After this change, we can specify NULL values as above.